### PR TITLE
Make modules container collapsable 

### PR DESCRIPTION
**Pull request by bp323**
- [x] Add 'collapse' icon (`fa-angle-double-left`)
- [x] Add collapse functionality
- [x] Check responsiveness 

screen shot 2015-03-23 at 11 49 17

### CODE REVIEW

**Comment by bp323** [comment]
Reassigning to @Coenego for review.

File: shared/gh/js/views/gh.admin-batch-edit.js [hunk 1]
```diff
@@ -444,27 +444,54 @@ define(['gh.core', 'gh.constants', 'moment', 'gh.calendar', 'gh.admin-event-type
     };
 
     /**
+     * Calculate the width of the batch edit header header and return the result in pixels
+     *
```
Comment (Coenego) [inline shared/gh/js/views/gh.admin-batch-edit.js]: `@return` missing

File: shared/gh/css/gh.base.css [hunk 2]
```diff
@@ -235,6 +259,74 @@ select[disabled],
     padding-left: 320px;
     width: 100%;
     height: 100%;
```
Comment (Coenego) [inline shared/gh/css/gh.base.css]: `height` should go before `padding-left`

**Comment by Coenego** [comment]
Some minor comments left. Back to @bp323 for follow-up

**Comment by bp323** [comment]
Back to @Coenego

**Pull request merged**